utils/frame/rpc/support/src/lib.rs

```diff
@@ -54,7 +52,7 @@ use sc_rpc_api::state::StateClient;
 /// # struct TestRuntime;
 /// #
 /// # decl_module! {
-///	#     pub struct Module<T: Config> for enum Call where origin: T::Origin {}
+/// 	#     pub struct Module<T: Config> for enum Call where origin: T::Origin {}
```

Comment:
```suggestion
/// #     pub struct Module<T: Config> for enum Call where origin: T::Origin {}
```

something strange with the spacing here before

Reply:
Yeah, but that was weird before the formatting. I've fixed it now I think.